gatsby/installing.rst

```diff
@@ -33,13 +33,13 @@ The CLI allows to initialize a project:
    gatsby new hello-world
 
 This command says gatsby-cli to create a new Gatsby project with name `hello-world` with a basic default file structure.
```

Comment:
s/says/tells

"with the name"

Reply:
Remove "simple" that is not needed and adds no value